src/Applications/LDAS_App/lenkf.j.template

```diff
@@ -732,6 +735,11 @@ EOF
 ## End of the while ( $counter <= ${NUM_SGMT} ) loop ##
 end
 
+if ($rc != 0) then
```

Comment:
If we drop the exit statement above, this if block should be moved to ~line 714, right?